stories/hits.stories.js

```diff
@@ -105,4 +107,39 @@ storiesOf('Hits', module)
         })
       );
     })
+  )
+  .add(
+    'with insights helper',
+    withHits(
+      ({ search, container, instantsearch }) => {
+        search.addWidget(
+          instantsearch.widgets.configure({
+            attributesToSnippet: ['name', 'description'],
+          })
+        );
+
+        search.addWidget(
+          instantsearch.widgets.hits({
+            container,
+            templates: {
+              item: item => `
+            <h4>${item.name}</h4>
+            <button
+              ${insights('clickedObjectIDsAfterSearch', {
+                objectIDs: [item.objectID],
+                eventName: 'Add to cart',
+              })} >Add to cart</button>
+            `,
+            },
+          })
+        );
+      },
+      {
+        searchParameters: {
+          clickAnalytics: true,
```

Comment:
Since we use `configure` above, can't we set `clickAnalytics` there?

Reply:
fixed here: 0abc39ee4ba7ba9af9443e1eb032cb3cb577012a